### PR TITLE
[zh-cn]sync scheduling-eviction/topology-spread-constraints.md

### PR DESCRIPTION
**Pull request by my-git9**
[zh-cn]sync scheduling-eviction/topology-spread-constraints.md

### CODE REVIEW

File: content/zh-cn/docs/concepts/scheduling-eviction/topology-spread-constraints.md [hunk 1]
```diff
@@ -276,13 +276,12 @@ your cluster. Those fields are:
...
-  `NodeInclusionPolicyInPodTopologySpread` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
-  in order to use it.
+  The `nodeAffinityPolicy` is an alpha-level field added in 1.25. You can disable it by disabling the
+  `NodeInclusionPolicyInPodTopologySpread` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/).
   -->
   `nodeAffinityPolicy` 是 1.25 中新增的一个 Alpha 级别字段。
-  你必须启用 `NodeInclusionPolicyInPodTopologySpread`
-  [特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)才能使用此字段。
+  你可以通过禁用 `NodeInclusionPolicyInPodTopologySpread`
+  [特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)来关闭此此特性。
```
Comment (mengjiao-liu) [inline content/zh-cn/docs/concepts/scheduling-eviction/topology-spread-constraints.md:284]:
```suggestion
  [特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)来禁用此特性。
```

File: content/zh-cn/docs/concepts/scheduling-eviction/topology-spread-constraints.md [hunk 2]
```diff
@@ -165,10 +165,10 @@ your cluster. Those fields are:
   {{< note >}}
   <!--
   The `minDomains` field is a beta field and enabled by default in 1.25. You can disable it by disabling the
-  `MinDomainsInPodToplogySpread` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/).
+  `MinDomainsInPodTopologySpread` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/).
   -->
   `minDomains` 字段是一个 Alpha 字段，在 1.25 中默认被启用。
-  你可以通过禁用 `MinDomainsInPodToplogySpread`
+  你可以通过禁用 `MinDomainsInPodTopologySpread`
   [特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)来禁用该字段。
```
Comment (mengjiao-liu) [inline content/zh-cn/docs/concepts/scheduling-eviction/topology-spread-constraints.md:172]: upstream main branch have updated the content. Please resync the file.
```
./scripts/lsync.sh content/zh-cn/docs/concepts/scheduling-eviction/topology-spread-constraints.m
```

see https://github.com/kubernetes/website/pull/38566

File: content/zh-cn/docs/concepts/scheduling-eviction/topology-spread-constraints.md [hunk 3]
```diff
@@ -302,13 +301,12 @@ your cluster. Those fields are:
...
-  in order to use it.
+  The `nodeTaintsPolicy` is a beta-level field and enabled by default in 1.26. You can disable it by disabling the
+  `NodeInclusionPolicyInPodTopologySpread` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/).
   -->
-  `nodeTaintsPolicy` 是 1.25 中新增的一个 Alpha 级别字段。
-  你必须启用 `NodeInclusionPolicyInPodTopologySpread`
-  [特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)才能使用此字段。
+  `nodeTaintsPolicy` 是一个 Beta 级别字段，在 1.26 版本默认启用。
+  你可以通过禁用 `NodeInclusionPolicyInPodTopologySpread`
+  [特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)来关闭此特性。
```
Comment (mengjiao-liu) [inline content/zh-cn/docs/concepts/scheduling-eviction/topology-spread-constraints.md:309]:
```suggestion
  [特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)来禁用此特性。
```

**Review by mengjiao-liu — CHANGES_REQUESTED** [review]

File: content/zh-cn/docs/concepts/scheduling-eviction/topology-spread-constraints.md [hunk 4]
```diff
@@ -164,12 +164,12 @@ your cluster. Those fields are:
...
   <!--
-  The `minDomains` field is a beta field and enabled by default in 1.25. You can disable it by disabling the
-  `MinDomainsInPodToplogySpread` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/).
+  The `minDomains` field is a beta field and disabled by default in 1.25. You can enable it by enabling the
+  `MinDomainsInPodTopologySpread` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/).
   -->
-  `minDomains` 字段是一个 Alpha 字段，在 1.25 中默认被启用。
-  你可以通过禁用 `MinDomainsInPodToplogySpread`
-  [特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)来禁用该字段。
+  `minDomains` 字段是一个 Alpha 字段，在 1.25 中默认被禁用。
```
Comment (mengjiao-liu) [inline content/zh-cn/docs/concepts/scheduling-eviction/topology-spread-constraints.md:170]:
```suggestion
  `minDomains` 字段是一个 Beta 字段，在 1.25 中默认被禁用。
```

File: content/zh-cn/docs/concepts/scheduling-eviction/topology-spread-constraints.md [hunk 5]
```diff
@@ -276,13 +276,12 @@ your cluster. Those fields are:
...
-  `NodeInclusionPolicyInPodTopologySpread` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
-  in order to use it.
+  The `nodeAffinityPolicy` is an alpha-level field added in 1.25. You can disable it by disabling the
+  `NodeInclusionPolicyInPodTopologySpread` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/).
   -->
   `nodeAffinityPolicy` 是 1.25 中新增的一个 Alpha 级别字段。
-  你必须启用 `NodeInclusionPolicyInPodTopologySpread`
-  [特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)才能使用此字段。
+  你可以通过禁用 `NodeInclusionPolicyInPodTopologySpread`
+  [特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)来关禁用此特性。
```
Comment (mengjiao-liu) [inline content/zh-cn/docs/concepts/scheduling-eviction/topology-spread-constraints.md:284]:
```suggestion
  [特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)来禁用此字段。
```

File: content/zh-cn/docs/concepts/scheduling-eviction/topology-spread-constraints.md [hunk 6]
```diff
@@ -302,13 +301,12 @@ your cluster. Those fields are:
...
-  in order to use it.
+  The `nodeTaintsPolicy` is a beta-level field and enabled by default in 1.26. You can disable it by disabling the
+  `NodeInclusionPolicyInPodTopologySpread` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/).
   -->
-  `nodeTaintsPolicy` 是 1.25 中新增的一个 Alpha 级别字段。
-  你必须启用 `NodeInclusionPolicyInPodTopologySpread`
-  [特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)才能使用此字段。
+  `nodeTaintsPolicy` 是一个 Beta 级别字段，在 1.26 版本默认启用。
+  你可以通过禁用 `NodeInclusionPolicyInPodTopologySpread`
+  [特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)来禁用此特性。
```
Comment (mengjiao-liu) [inline content/zh-cn/docs/concepts/scheduling-eviction/topology-spread-constraints.md:309]:
```suggestion
  [特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)来禁用此字段。
```

**Review by mengjiao-liu — CHANGES_REQUESTED** [review]

**Comment by tengqm** [comment]
Thanks.
/approve

**Pull request merged**